platforms/documentation/docs/src/docs/userguide/dep-man/basics/centralizing_dependencies.adoc

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[centralizing-dependencies]]
+= 4. Centralizing dependencies
+
+Central dependencies can be managed in Gradle using various techniques such as platforms and version catalogs.
+Each approach offers its own advantages and helps in centralizing and managing dependencies efficiently.
+
+== Using platforms
+
+A platform is a set of dependency constraints designed to manage the transitive dependencies of a library or application.
+
+When you define a platform in Gradle, you're essentially specifying a set of dependencies that are meant to be used together, ensuring compatibility and simplifying dependency management:
+
+====
+include::sample[dir="snippets/java-platform/dep-man/kotlin",files="platform/build.gradle.kts[]"]
+include::sample[dir="snippets/java-platform/dep-man/groovy",files="platform/build.gradle[]"]
+====
+
+Then, you can use that platform in your project:
+
+====
+include::sample[dir="snippets/java-platform/dep-man/kotlin",files="app/build.gradle.kts[]"]
+include::sample[dir="snippets/java-platform/dep-man/groovy",files="app/build.gradle[]"]
+====
+
+Here, `platform` defines versions for `commons-lang3`, `guava`, and `slf4j-api`, ensuring they are compatible.
+
+Maven's BOM (Bill of Materials) is a popular type of platform that Gradle supports.
+A BOM file lists dependencies with specific versions, allowing you to manage these versions in a centralized way.
+
+A popular platform is the link:https://mvnrepository.com/artifact/io.spring.platform/platform-bom/Cairo-RELEASE[Spring IO Bill of Materials] which is maintained and hosted by link:https://spring.io/[Spring].
```

Comment:
We updated the below snippet to use `spring-boot-dependencies`, but this link still references the Spring IO BOM